src/skeptic/lib.rs

```diff
@@ -730,7 +730,7 @@ pub mod rt {
 
     pub fn compile_test(root_dir: &str, out_dir: &str, target_triple: &str, test_text: &str) {
         let rustc = &env::var("RUSTC").unwrap_or_else(|_| String::from("rustc"));
-        let outdir = &TempDir::new("rust-skeptic").unwrap();
```

Comment:
There is an interface that supports path prefixes if desired, but it's a tiny bit wordier: https://docs.rs/tempfile/3.1.0/tempfile/struct.TempDir.html#examples